### ISSUE TITLE
Does not work past the first dependency

### ISSUE THREAD

**Issue by memelet** [systeminsights/mrequire#1]
If a module A includes a module B that uses mrequire for dependencies C, the dependencies C are not found.

**Comment by wemrysi** [comment]
Yeah, bummer. `require.main` is global to the node process, not scoped to the module so I don't think this could ever work transitively.
